### PR TITLE
3.5.8 Changelog

### PR DESCRIPTION
**Pull request by burningtnt**

### CODE REVIEW

**Comment by burningtnt** [comment]
> Credit for MCBBS restored in https://github.com/HMCL-dev/HMCL/pull/2958. The related PR was written into the changelog when the relevant credit was removed before. Would you consider mentioning it this time?

For users who update there launcher frmo 3.5.7 to 3.5.8, the credit is never removed.

File: _changelog/stable.md [hunk 1]
```diff
@@ -10,7 +10,23 @@ toc: true
...
 *Notice: changelogs are written in Chinese.*
 
-<h1 id="nowchange">HMCL 3.5.7</h1>
+<h1 id="nowchange">HMCL 3.5.8</h1>
+
+- 启动器
+  - GP-2947: 更新模组和整合包翻译信息
+  - GP-2951: 优化了启动器长期运行后的性能表现
+  - GP-2942: 游戏版本的图标现在支持 JPG、GIF 等格式
+  - GP-2950: 启动器启动后，将自动加载同目录下的 font.ttf 或 font.otf
```
[the remark this answers is not in the document]
Comment (burningtnt) [reply _changelog/stable.md:19]: Does the web page of HMCL support the \`\` mark?

[the remark this answers is not in the document]
Comment (burningtnt) [reply _changelog/stable.md]: Added

**Comment by burningtnt** [comment]
> Maybe I did not make it clear enough. I mean, the related credits for MCBBS were indeed removed in 3.5.7 and mentioned in the changelog, while the related changes were partially restored in 3.5.8.

My fault. My memory is wrong.

**Review by zkitefly — APPROVED** [review]

File: _changelog/stable.md [hunk 2]
```diff
@@ -10,7 +10,24 @@ toc: true
...
-<h1 id="nowchange">HMCL 3.5.7</h1>
+<h1 id="nowchange">HMCL 3.5.8</h1>
+
+- 启动器
+  - GP-2947: 更新模组和整合包翻译信息
+  - GP-2951: 优化了启动器长期运行后的性能表现
+  - GP-2942: 游戏版本的图标现在支持 JPG、GIF 等格式
+  - GP-2950: 启动器启动后，将自动加载同目录下的 `font.ttf` 或 `font.otf`
+- 下载
+  - GP-2958: 还原对 MCBBS 的鸣谢。感谢你，MCBBS！
```
Comment (zkitefly) [inline _changelog/stable.md:21]: 感觉有点奇怪？🤔

Comment (burningtnt) [reply _changelog/stable.md:21]: 故意加的。不过把“感谢”更改为“谢谢”会好一点？

**Pull request merged**